dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -1,10 +1,12 @@
 import 'dart:async';
```

Comment:
The analyzer does not like that this file isn't licensed, but this is code generated. What do we do in this case? @HansMuller